src/drawable.hpp

```diff
@@ -21,13 +21,6 @@ class drawable
             uint16_t width, ///< Width of the display
             uint16_t height ///< Height of the display
   );
-  ~drawable ();
```

Comment:
are  you sure we shouldn't write something instead of deleting everything

Reply:
sure. It will use just use defaults and defaults fit our case